### PR TITLE
Style trivia screen

### LINKED ISSUE #38: Trivia Screen
### Description

Display basic trivia about a movie on a screen. The player is redirected here after either the failure screen or the success screen.

### Tasks

- [x] The screen displays metadata about the movie in question (e.g., genre, release year, popularity, ...)
- [x] The screen has a button that will allow the player to continue with a new question (i.e., redirects to [Blurred Poster Screen](https://github.com/kwizapp/kwiz/issues/9)).

### Context 
[Story: Trivia Screen](https://github.com/kwizapp/kwiz/issues/14)

### PR DESCRIPTION
**Pull request by alexscheitlin**
### Description

Implement a basic trivia screen.

image

### Related Issues

#38 
https://github.com/kwizapp/kwiz/issues/14

### Checklist

- [x] Have you added tests where necessary? Do all the test pass? 
- [x] Have you added descriptive comments to your code?
- [x] Have you updated the documentation related to this propo

### CODE REVIEW

File: src/pages/Trivia.tsx [hunk 1]
```diff
@@ -1,20 +1,96 @@
...
 const Trivia = () => {
   const history = useHistory()
 
+  const { currentImdbId } = useStore()
+
+  // FIX ME: I'm returning random movies instead of the specified one!
+  // `currentImdbId` is always empty
+  const { loading, error, data } = useQuery<MovieData>(MOVIE, {
+    variables: { imdbId: currentImdbId },
+  })
```
Comment (alexscheitlin) [inline src/pages/Trivia.tsx]: Needs some fixing

Comment (alexscheitlin) [reply src/pages/Trivia.tsx]: Fixed

**Review by nikzaugg — APPROVED** [review]
Looks good. Thanks.

Should we create a new issue to extend this? Right now it does not show any additional information other than the title, right?

Or what is the plan here? @rschlaefli @alexscheitlin

**Comment by alexscheitlin** [comment]
> Looks good. Thanks.
> 
> Should we create a new issue to extend this? Right now it does not show any additional information other than the title, right?
> 
> Or what is the plan here? @rschlaefli @alexscheitlin

Yes, we should. First we'd need to have a look at what information can be provided by the metadata-service and then include them in the nest-api.

**Pull request merged**